### ISSUE TITLE
Build fails on Arch

### ISSUE THREAD

**Issue by JustCryen** [trackmastersteve/alienfx#103]
### 1. Expected behavior.

- Manual installation succeeds

### 2. Actual behavior.

- Manual installation fails

### 3. Steps to reproduce the problem.

- I followed the manual installation of the 2.3.5 release since it's not yet updated on the AUR
- The manual installation worked for me some time ago when I reported the zone scanner issue on m15, since then I stopped using this program but now I wanted to check what's new on the new update.

### Logs

```
❯ sudo python3 setup.py install
running install
/usr/lib/python3.10/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
/usr/lib/python3.10/site-packages/setuptools/command/easy_install.py:156: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
/usr/lib/python3.10/site-packages/pkg_resources/__init__.py:116: PkgResourcesDeprecationWarning: 2.0.5-build-libtorrent-rasterbar-src-libtorrent-rasterbar-2.0.5-bindings-python is an invalid version and will not be supported in a future release
  warnings.warn(
running bdist_egg
running egg_info
creating alienfx.egg-info
writing alienfx.egg-info/PKG-INFO
writing dependency_links to alienfx.egg-info/dependency_links.txt
writing entry points to alienfx.egg-info/entry_points.txt
writing requirements to alienfx.egg-info/requires.txt
writing top-level names to alienfx.egg-info/top_level.txt
writing manifest file 'alienfx.egg-info/SOURCES.txt'
listing git files failed - pretending there aren't any
reading manifest file 'alienfx.egg-info/SOURCES.txt'
adding license file 'LICENSE'
writing manifest file 'alienfx.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-x86_64/egg
running install_lib
running build_py
creating build
creating build/lib
creating build/lib/alienfx
copying alienfx/__init__.py -> build/lib/alienfx
copying alienfx/common.py -> build/lib/alienfx
creating build/lib/alienfx/ui
copying alienfx/ui/__init__.py -> build/lib/alienfx/ui
creating build/lib/alienfx/core
copying alienfx/core/controller_m17x.py -> build/lib/alienfx/core
copying alienfx/core/controller_m14xr1.py -> build/lib/alienfx/core
copying alienfx/core/controller_aurora.py -> build/lib/alienfx/core
copying alienfx/core/controller_m13xr2.py -> build/lib/alienfx/core
copying alienfx/core/controller_m11xr3.py -> build/lib/alienfx/core
copying alienfx/core/controller.py -> build/lib/alienfx/core
copying alienfx/core/controller_area51.py -> build/lib/alienfx/core
copying alienfx/core/controller_m13xr3.py -> build/lib/alienfx/core
copying alienfx/core/usbdriver.py -> build/lib/alienfx/core
copying alienfx/core/controller_m11xr1.py -> build/lib/alienfx/core
copying alienfx/core/cmdpacket.py -> build/lib/alienfx/core
copying alienfx/core/controller_m17xr4.py -> build/lib/alienfx/core
copying alienfx/core/themefile.py -> build/lib/alienfx/core
copying alienfx/core/controller_m18xr2.py -> build/lib/alienfx/core
copying alienfx/core/prober.py -> build/lib/alienfx/core
copying alienfx/core/logger.py -> build/lib/alienfx/core
copying alienfx/core/zonescanner.py -> build/lib/alienfx/core
copying alienfx/core/__init__.py -> build/lib/alienfx/core
copying alienfx/core/controller_17r1.py -> build/lib/alienfx/core
copying alienfx/core/controller_m11xr2.py -> build/lib/alienfx/core
copying alienfx/core/controller_m14xr3.py -> build/lib/alienfx/core
copying alienfx/core/controller_m17xr3.py -> build/lib/alienfx/core
copying alienfx/core/controller_17r3.py -> build/lib/alienfx/core
copying alienfx/core/controller_m14xr2.py -> build/lib/alienfx/core
copying alienfx/core/controller_m15x.py -> build/lib/alienfx/core
copying alienfx/core/controller_area51_r2.py -> build/lib/alienfx/core
creating build/lib/alienfx/ui/console
copying alienfx/ui/console/__init__.py -> build/lib/alienfx/ui/console
copying alienfx/ui/console/main.py -> build/lib/alienfx/ui/console
creating build/lib/alienfx/ui/gtkui
copying alienfx/ui/gtkui/colour_palette.py -> build/lib/alienfx/ui/gtkui
copying alienfx/ui/gtkui/gtkui.py -> build/lib/alienfx/ui/gtkui
copying alienfx/ui/gtkui/action_renderer.py -> build/lib/alienfx/ui/gtkui
copying alienfx/ui/gtkui/__init__.py -> build/lib/alienfx/ui/gtkui
creating build/lib/alienfx/ui/gtkui/glade
copying alienfx/ui/gtkui/glade/ui.glade -> build/lib/alienfx/ui/gtkui/glade
creating build/lib/alienfx/data
creating build/lib/alienfx/data/icons
creating build/lib/alienfx/data/icons/hicolor
creating build/lib/alienfx/data/icons/hicolor/scalable
creating build/lib/alienfx/data/icons/hicolor/scalable/apps
copying alienfx/data/icons/hicolor/scalable/apps/alienfx.svg -> build/lib/alienfx/data/icons/hicolor/scalable/apps
creating build/lib/alienfx/data/themes
copying alienfx/data/themes/default.json -> build/lib/alienfx/data/themes
creating build/lib/alienfx/data/etc
creating build/lib/alienfx/data/etc/udev
creating build/lib/alienfx/data/etc/udev/rules.d
copying alienfx/data/etc/udev/rules.d/10-alienfx.rules -> build/lib/alienfx/data/etc/udev/rules.d
creating build/bdist.linux-x86_64
creating build/bdist.linux-x86_64/egg
creating build/bdist.linux-x86_64/egg/alienfx
creating build/bdist.linux-x86_64/egg/alienfx/data
creating build/bdist.linux-x86_64/egg/alienfx/data/icons
creating build/bdist.linux-x86_64/egg/alienfx/data/icons/hicolor
creating build/bdist.linux-x86_64/egg/alienfx/data/icons/hicolor/scalable
creating build/bdist.linux-x86_64/egg/alienfx/data/icons/hicolor/scalable/apps
copying build/lib/alienfx/data/icons/hicolor/scalable/apps/alienfx.svg -> build/bdist.linux-x86_64/egg/alienfx/data/icons/hicolor/scalable/apps
creating build/bdist.linux-x86_64/egg/alienfx/data/etc
creating build/bdist.linux-x86_64/egg/alienfx/data/etc/udev
creating build/bdist.linux-x86_64/egg/alienfx/data/etc/udev/rules.d
copying build/lib/alienfx/data/etc/udev/rules.d/10-alienfx.rules -> build/bdist.linux-x86_64/egg/alienfx/data/etc/udev/rules.d
creating build/bdist.linux-x86_64/egg/alienfx/data/themes
copying build/lib/alienfx/data/themes/default.json -> build/bdist.linux-x86_64/egg/alienfx/data/themes
creating build/bdist.linux-x86_64/egg/alienfx/ui
creating build/bdist.linux-x86_64/egg/alienfx/ui/console
copying build/lib/alienfx/ui/console/__init__.py -> build/bdist.linux-x86_64/egg/alienfx/ui/console
copying build/lib/alienfx/ui/console/main.py -> build/bdist.linux-x86_64/egg/alienfx/ui/console
creating build/bdist.linux-x86_64/egg/alienfx/ui/gtkui
copying build/lib/alienfx/ui/gtkui/colour_palette.py -> build/bdist.linux-x86_64/egg/alienfx/ui/gtkui
copying build/lib/alienfx/ui/gtkui/gtkui.py -> build/bdist.linux-x86_64/egg/alienfx/ui/gtkui
copying build/lib/alienfx/ui/gtkui/action_renderer.py -> build/bdist.linux-x86_64/egg/alienfx/ui/gtkui
creating build/bdist.linux-x86_64/egg/alienfx/ui/gtkui/glade
copying build/lib/alienfx/ui/gtkui/glade/ui.glade -> build/bdist.linux-x86_64/egg/alienfx/ui/gtkui/glade
copying build/lib/alienfx/ui/gtkui/__init__.py -> build/bdist.linux-x86_64/egg/alienfx/ui/gtkui
copying build/lib/alienfx/ui/__init__.py -> build/bdist.linux-x86_64/egg/alienfx/ui
creating build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_m17x.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_m14xr1.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_aurora.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_m13xr2.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_m11xr3.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_area51.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_m13xr3.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/usbdriver.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_m11xr1.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/cmdpacket.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_m17xr4.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/themefile.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_m18xr2.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/prober.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/logger.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/zonescanner.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/__init__.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_17r1.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_m11xr2.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_m14xr3.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_m17xr3.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_17r3.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_m14xr2.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_m15x.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/core/controller_area51_r2.py -> build/bdist.linux-x86_64/egg/alienfx/core
copying build/lib/alienfx/__init__.py -> build/bdist.linux-x86_64/egg/alienfx
copying build/lib/alienfx/common.py -> build/bdist.linux-x86_64/egg/alienfx
byte-compiling build/bdist.linux-x86_64/egg/alienfx/ui/console/__init__.py to __init__.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/ui/console/main.py to main.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/ui/gtkui/colour_palette.py to colour_palette.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/ui/gtkui/gtkui.py to gtkui.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/ui/gtkui/action_renderer.py to action_renderer.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/ui/gtkui/__init__.py to __init__.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/ui/__init__.py to __init__.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_m17x.py to controller_m17x.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_m14xr1.py to controller_m14xr1.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_aurora.py to controller_aurora.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_m13xr2.py to controller_m13xr2.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_m11xr3.py to controller_m11xr3.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller.py to controller.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_area51.py to controller_area51.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_m13xr3.py to controller_m13xr3.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/usbdriver.py to usbdriver.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_m11xr1.py to controller_m11xr1.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/cmdpacket.py to cmdpacket.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_m17xr4.py to controller_m17xr4.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/themefile.py to themefile.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_m18xr2.py to controller_m18xr2.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/prober.py to prober.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/logger.py to logger.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/zonescanner.py to zonescanner.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/__init__.py to __init__.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_17r1.py to controller_17r1.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_m11xr2.py to controller_m11xr2.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_m14xr3.py to controller_m14xr3.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_m17xr3.py to controller_m17xr3.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_17r3.py to controller_17r3.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_m14xr2.py to controller_m14xr2.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_m15x.py to controller_m15x.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/core/controller_area51_r2.py to controller_area51_r2.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/__init__.py to __init__.cpython-310.pyc
byte-compiling build/bdist.linux-x86_64/egg/alienfx/common.py to common.cpython-310.pyc
installing package data to build/bdist.linux-x86_64/egg
running install_data
creating build/bdist.linux-x86_64/egg/share
creating build/bdist.linux-x86_64/egg/share/applications
copying alienfx/data/share/applications/alienfx.desktop -> build/bdist.linux-x86_64/egg/share/applications
creating build/bdist.linux-x86_64/egg/share/icons
creating build/bdist.linux-x86_64/egg/share/icons/hicolor
creating build/bdist.linux-x86_64/egg/share/icons/hicolor/scalable
creating build/bdist.linux-x86_64/egg/share/icons/hicolor/scalable/apps
copying alienfx/data/icons/hicolor/scalable/apps/alienfx.svg -> build/bdist.linux-x86_64/egg/share/icons/hicolor/scalable/apps
creating build/bdist.linux-x86_64/egg/share/icons/hicolor/48x48
creating build/bdist.linux-x86_64/egg/share/icons/hicolor/48x48/apps
copying alienfx/data/icons/hicolor/48x48/apps/alienfx.png -> build/bdist.linux-x86_64/egg/share/icons/hicolor/48x48/apps
creating build/bdist.linux-x86_64/egg/share/pixmaps
copying alienfx/data/pixmaps/alienfx.png -> build/bdist.linux-x86_64/egg/share/pixmaps
creating build/bdist.linux-x86_64/egg/share/man
creating build/bdist.linux-x86_64/egg/share/man/man1
copying docs/man/alienfx.1 -> build/bdist.linux-x86_64/egg/share/man/man1
creating build/bdist.linux-x86_64/egg/EGG-INFO
copying alienfx.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO
copying alienfx.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying alienfx.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying alienfx.egg-info/entry_points.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying alienfx.egg-info/requires.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying alienfx.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
zip_safe flag not set; analyzing archive contents...
creating dist
creating 'dist/alienfx-2.3.5-py3.10.egg' and adding 'build/bdist.linux-x86_64/egg' to it
removing 'build/bdist.linux-x86_64/egg' (and everything under it)
Processing alienfx-2.3.5-py3.10.egg
Copying alienfx-2.3.5-py3.10.egg to /usr/lib/python3.10/site-packages
Adding alienfx 2.3.5 to easy-install.pth file
Installing alienfx script to /usr/bin
Installing alienfx-gtk script to /usr/bin

Installed /usr/lib/python3.10/site-packages/alienfx-2.3.5-py3.10.egg
Processing dependencies for alienfx==2.3.5
Searching for pyusb==1.2.1
Best match: pyusb 1.2.1
Adding pyusb 1.2.1 to easy-install.pth file

Using /usr/lib/python3.10/site-packages
Finished processing dependencies for alienfx==2.3.5
❯ sudo python3 setup.py install_data
running install_data
/usr/lib/python3.10/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
copying alienfx/data/share/applications/alienfx.desktop -> /usr/share/applications
copying alienfx/data/icons/hicolor/scalable/apps/alienfx.svg -> /usr/share/icons/hicolor/scalable/apps
copying alienfx/data/icons/hicolor/48x48/apps/alienfx.png -> /usr/share/icons/hicolor/48x48/apps
copying alienfx/data/pixmaps/alienfx.png -> /usr/share/pixmaps
copying docs/man/alienfx.1 -> /usr/share/man/man1
❯ alienfx
Traceback (most recent call last):
  File "/usr/bin/alienfx", line 33, in <module>
    sys.exit(load_entry_point('alienfx==2.3.5', 'console_scripts', 'alienfx')())
  File "/usr/bin/alienfx", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 171, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/usr/lib/python3.10/site-packages/alienfx-2.3.5-py3.10.egg/alienfx/ui/console/__init__.py", line 2, in <module>
  File "/usr/lib/python3.10/site-packages/alienfx-2.3.5-py3.10.egg/alienfx/ui/console/main.py", line 33, in <module>
  File "/usr/lib/python3.10/site-packages/alienfx-2.3.5-py3.10.egg/alienfx/core/prober.py", line 47, in <module>
ModuleNotFoundError: No module named 'alienfx.core.controller_13r3'
```

### 4. Specifications.

- Project Version: 2.3.5
- Python Version: 3.10.4-1
- Operating System: ArchLinux
- Hardware Version: Alienware m15

**Comment by JustCryen** [comment]
Well, Python on my system probably committed suicide, this might explain most of what happened above…
Damn, I hate python…

**Comment by jmastertrainer** [comment]
I'm having the same issue on Ubuntu. Did you find a solution? 

$ alienfx
Traceback (most recent call last):
  File "/usr/local/bin/alienfx", line 33, in <module>
    sys.exit(load_entry_point('alienfx==2.3.5', 'console_scripts', 'alienfx')())
  File "/usr/local/bin/alienfx", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/lib/python3.10/importlib/metadata/__init__.py", line 171, in load
    module = import_module(match.group('module'))
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/usr/local/lib/python3.10/dist-packages/alienfx-2.3.5-py3.10.egg/alienfx/ui/console/__init__.py", line 2, in <module>
  File "/usr/local/lib/python3.10/dist-packages/alienfx-2.3.5-py3.10.egg/alienfx/ui/console/main.py", line 33, in <module>
  File "/usr/local/lib/python3.10/dist-packages/alienfx-2.3.5-py3.10.egg/alienfx/core/prober.py", line 47, in <module>
ModuleNotFoundError: No module named 'alienfx.core.controller_13r3'

**Comment by JustCryen** [comment]
pip overwritten some of the dependencies and they couldn't be tracked by pacman.
I removed them from pip and tried reinstalling them with pacman.
This should've worked without any issues but for some reason pip left some garbage behind and pacman still couldn't find the owner to some files so… I did a really nasty thing and removed them manually and tried installing the dependencies again with pacman.
It did work but remember to backup the files you're about to remove manually just to be sure…
